linux_os/guide/services/obsolete/telnet/service_telnet_disabled/rule.yml

```diff
@@ -5,36 +5,15 @@ prodtype: alinux2,ol7,ol8,ol9,rhel7,rhel8,rhel9,rhv4,sle15
 title: 'Disable telnet Service'
 
 description: |-
-    The <tt>telnet</tt> service configuration file <tt>/etc/xinetd.d/telnet</tt>
-    is not created automatically. If it was created manually, check the
-    <tt>/etc/xinetd.d/telnet</tt> file and ensure that <tt>disable = no</tt>
-    is changed to read <tt>disable = yes</tt> as follows below:
-    <pre>
-    # description: The telnet server serves telnet sessions; it uses \\
-    #       unencrypted username/password pairs for authentication.
-    service telnet
-    {
-            flags           = REUSE
-            socket_type     = stream
-
-            wait            = no
-            user            = root
-            server          = /usr/sbin/in.telnetd
-            log_on_failure  += USERID
-            disable         = yes
-    }
-    </pre>
-    If the <tt>/etc/xinetd.d/telnet</tt> file does not exist, make sure that
-    the activation of the <tt>telnet</tt> service on system boot is disabled
-    via the following command:
-    {{{ describe_socket_disable(socket="rexec") }}}
+    Make sure that the activation of the <tt>telnet</tt> service on system boot is disabled via
+    the following command:
```

Comment:
Justa  nitpick - when combined with the macron on the next line, the "following command" phrase is there twice. Try to compile te content and look in the HTML guide.

Reply:
Hi @vojtapolasek , this was probably there even before the changes in this PR. In any case, let me check and fix. Thanks for noticing this.

Reply:
It is fixed now.